x/blob/types/payforblob_test.go

```diff
@@ -176,7 +176,7 @@ func TestValidateBasic(t *testing.T) {
 		{
 			name:    "parity shares namespace id",
 			msg:     paritySharesMsg,
-			wantErr: ErrParitySharesNamespace,
```

Comment:
Can this error still occur. Do we have tests to ensure it errors as expected?

Reply:
It can no longer occur b/c `ErrInvalidNamespace` will be hit before `ErrParitySharesNamespace` is hit. I choose not to remove b/c a check exists for it [here](https://github.com/celestiaorg/celestia-app/blob/e03617e561fb3c01e33bf03859f53a05645a469d/x/blob/types/payforblob.go#L141-L143)